src/platforms/php/guides/laravel/configuration/laravel-options.mdx

```diff
@@ -4,47 +4,48 @@ sidebar_order: 1
 description: "Learn about Sentry's integration with Laravel and its options for breadcrumbs, and performance."
 ---
 
-All configuration for Laravel is done in `config/sentry.php`.
+Most configuration for Laravel is done in your `.env` file.
+You can also add additonal config options to your `config/sentry.php` file.
```

Comment:
Can we clarify what the difference is between configuration that goes into .env vs config/sentry.php? And how users know which bits go in each?

Reply:
By default, all settings in `config/sentry.php` are controlled by the environment variables set in the `.env` file.
The remaining options (https://docs.sentry.io/platforms/php/guides/laravel/configuration/options/) can be added manually to `config/sentry.php`.